src/ObjectUtils/PdbFileLLVM.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef OBJECT_UTILS_PDB_FILE_LLVM_H_
+#define OBJECT_UTILS_PDB_FILE_LLVM_H_
+
+#include <llvm/DebugInfo/CodeView/CVSymbolVisitor.h>
+#include <llvm/DebugInfo/CodeView/CodeView.h>
+#include <llvm/DebugInfo/CodeView/GUID.h>
+#include <llvm/DebugInfo/CodeView/SymbolDeserializer.h>
+#include <llvm/DebugInfo/CodeView/SymbolVisitorCallbackPipeline.h>
+#include <llvm/DebugInfo/CodeView/SymbolVisitorCallbacks.h>
+#include <llvm/DebugInfo/MSF/MappedBlockStream.h>
+#include <llvm/DebugInfo/PDB/Native/DbiStream.h>
+#include <llvm/DebugInfo/PDB/Native/ModuleDebugStream.h>
+#include <llvm/DebugInfo/PDB/Native/NativeSession.h>
+#include <llvm/DebugInfo/PDB/Native/PDBFile.h>
+#include <llvm/DebugInfo/PDB/PDB.h>
+#include <llvm/DebugInfo/PDB/PDBSymbolExe.h>
+#include <llvm/DebugInfo/PDB/PDBTypes.h>
+#include <llvm/Demangle/Demangle.h>
+
+#include <array>
+#include <filesystem>
+
+#include "ObjectUtils/PdbFile.h"
+#include "ObjectUtils/WindowsBuildIdUtils.h"
+#include "OrbitBase/Result.h"
+#include "symbol.pb.h"
+
+namespace orbit_object_utils {
+
+class PdbFileLLVM : public PdbFile {
```

Comment:
Nit: Per our naming conventions, this should be PdbFileLlvm, also for the filename (and same for the .cpp)

Reply:
Yep, done, thanks.